### ISSUE TITLE
[Request] Option to only auto sell gear from before BfA.

### ISSUE THREAD

**Issue by Donhorn** [Jaliborc/Scrap#40]
Could we have an option to only auto sell equipment from before BfA? I want to hold onto the much weaker BfA gear for scrapping into crafting mats, but don't want to manually sell all the junk from running old raids. Giving us a Pre-BfA only option or allowing us to manually set the ilvl we want to start keeping gear would be wonderful.

**Comment by martijnhartlief** [comment]
I agree this would be amazing.

**Comment by BradCOnGitHub** [comment]
Or at least let me set the ilvl for "Low Equipment" since it keeps picking stuff I don't really want sold yet.

**Comment by MartinBayard** [comment]
I agree, it would be fantastic !!

**Comment by BarbokOL** [comment]
Agree, this is what I got the addon for, would be nice if we could specify to sell equip under specific item level.
